mappings/net/minecraft/container/LoomContainer.mapping

```diff
@@ -33,3 +33,4 @@ CLASS ats net/minecraft/container/LoomContainer
 	METHOD e getCraftingResultSlotIndex ()I
 	METHOD f getCraftingWidth ()I
 	METHOD g getCraftingHeight ()I
+	METHOD h craftingSlotCount ()I
```

Comment:
getCraftingSlotCount?

Reply:
It's used by RecipeBookGui to detect when any of the crafting or crafting result slots is clicked on so it can hide the ghost item overlay that it displays whenever you select a recipe from the book that you lack the ingredients for.

Reply:
If CraftingContainer had documentation, it would say that the first N slots of the container are considered "crafting slots" that can have said overlay applied to them, where N is the number returned by getCraftingSlotCount.

Reply:
From code observation, clearly a getter to me...